index.html

```diff
@@ -2368,6 +2373,98 @@ <h3>
           can get their games and apps rated with IARC</a>.
         </p>
       </section>
+      <section>
+        <h3>
+          <code>shortcuts</code> member
+        </h3>
+        <p>
+          The <dfn>shortcuts</dfn> member is an <a>array</a> of
+          <a>ShortcutItem</a>s that provide access to key tasks within a web
+          application. For example, they can be used to link directly to a
+          user’s timeline within a social media application or their recent
```

Comment:
nit: There's a fancy apostrophe here; I think this should just be a regular ASCII apostrophe (maybe search the document for other instances of this).